runtime/src/jvmTest/kotlin/pbandk/UtilTest.kt

```diff
@@ -0,0 +1,61 @@
+package pbandk
+
+import org.junit.Test
+import pbandk.internal.Util
+import pbandk.wkt.Timestamp
+import kotlin.test.assertEquals
+
+class UtilTest {
```

Comment:
Nothing in these tests is JVM specific. So it'd be good to move these into `commonTest` since it'll be nice to have the same date formatting verification happening for all platforms.